examples/openmm/metad/alanine-dipeptide.py

```diff
@@ -113,11 +113,11 @@ def main(argv=[]):
     callback = MetaDLogger(hills_file, stride) if args.log else None
 
     tic = time.perf_counter()
-    pysages.run(method, generate_simulation, timesteps, callback)
+    run_result = pysages.run(method, generate_simulation, timesteps, callback)
     toc = time.perf_counter()
     print(f"Completed the simulation in {toc - tic:0.4f} seconds.")
 
```

Comment:
Maybe we can include plotting the results in this example too?

Reply:
This is a good point. I can include steps to plot the results in this file too.

Reply:
Thanks!

Reply:
Done.